### PR TITLE
Ben/docu 750

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

File: docs/configure-storage.md [hunk 1]
```diff
@@ -94,9 +94,9 @@ Although present when invoking the `help` command, Local Filesystem functionalit
 |:---|:---|
 | [`filesystem add adls2 oauth`](./command-reference.md#filesystem-add-adls2-oauth) | Add an ADLS Gen 2 file system resource using a service principal and oauth credentials |
 | [`filesystem add adls2 sharedKey`](./command-reference.md#filesystem-add-adls2-sharedkey) | Add an ADLS Gen 2 file system resource using access key credentials |
-| [`filesystem add gcs`](./command-reference.md#filesystem-add-gcs) | Add a Google Cloud Storage file system resource |
-| [`filesystem add hdfs`](./command-reference.md#filesystem-add-hdfs) | Add a Hadoop HDFS file system resource |
-| [`filesystem add s3a`](./command-reference.md#filesystem-add-s3a) | Add a S3 file system resource (choose this when using IBM COS) |
+| [`filesystem add gcs`](./command-reference.md#filesystem-add-gcs) | Add an Google Cloud Storage file system resource |
```
Comment (lucasdigital) [inline docs/configure-storage.md:97]: Add "a" Google Cloud Storage, surely?

File: docs/command-reference.md [hunk 2]
```diff
@@ -1587,7 +1587,7 @@ OPTIONS
 #### Mandatory Parameters
 
 :::info
-The Azure hive agent requires a ADLS Gen2 storage account and container name, this is only for the purposes of generating the correct location for the database. The container will not be accessed by the Hive agent and no data will be written to the container.
+The Azure hive agent requires a ADLS Gen2 storage account and container name, this is only for the purposes of generating the correct location for the metadata. The container will not be accessed by the agent and no data will be written to the container.
```
Comment (lucasdigital) [inline docs/command-reference.md:1359]: That sentence feels like it is repeating "The container" is unnecessary. How about "The agent won't access the container, and no data will be written to it."

File: docs/command-reference.md [hunk 3]
```diff
@@ -205,7 +205,7 @@ filesystem add gcs --file-system-id gcsAgent --bucket-name myGcsBucket --service
 
 Add a Hadoop Distributed File System as either a migration source or target using the `filesystem add hdfs` command.
 
-Creating an HDFS file system resource with this command will normally only be used when migrating to a target HDFS (rather than another storage service like ADLS Gen 2 or S3a). LiveData Migrator will attempt to auto-detect the *source* HDFS file system when started from the command line unless Kerberos is enabled on your source environment.
+Creating an HDFS resource with this command will normally only be used when migrating to a target HDFS (rather than another storage service like ADLS Gen 2 or S3a). LiveData Migrator will attempt to auto-detect the *source* HDFS when started from the command line unless Kerberos is enabled on your source environment.
```
Comment (lucasdigital) [inline docs/command-reference.md:208]: For consistency, should that be "used when migrating to a target HDFS **storage**"? While it sort of sounds okay to migrate to a file system, you go on to compare with another storage service.

**Review by lucasdigital — APPROVED** [review]
Just a couple of small suggestions.

File: docs/configure-storage.md (hunk 1 above)
Comment (benradstone) [reply docs/configure-storage.md:97]: Good catch

**Pull request merged**